gatsby_fp/src/components/Layout/Body/Fields/IntellCont/SubHeading.js

```diff
@@ -32,128 +32,94 @@ export default function IntellCont({ intellContList, publicationType }) {
     return sortedIntellContList
   }
 
-    const sortedIntellContList = sortStatusAndYear(intellContList)
+  const sortedIntellContList = sortStatusAndYear(intellContList)
+  const liInnerHTMLList =
+    sortedIntellContList.filter(element => (element.contype === publicationType || element.contypeother === publicationType)).map(element => {
+      let liString = '';
+      if (element.status) {
+        liString += `${element.status} `
+      }
+      if (element.contype && !element.contypeother) {
+        liString += `${element.contype} - `
+      }
+      if (element.contypeother) {
+        liString += `${element.contypeother} - `
+      }
+      element.intellcont_auth.forEach(elem => {
+        if (elem.lname) {
+          liString += `${elem.lname}, `
+        }
+        if (elem.fname && !elem.mname) {
+          liString += `${elem.fname}; `
+        }
+        if (elem.mname && !elem.fname) {
+          liString += `${elem.mname}.; `
+        }
+        if (elem.fname && elem.mname) {
+          liString += `${elem.fname}. ${elem.mname}.; `
+        }
+      })
+      if (element.status === "Published" && element.dty_pub) {
+        liString += `(${element.dty_pub}). `
+      }
+      if (element.status === "Accepted" && element.dty_acc) {
+        liString += `(${element.dty_acc}). `
+      }
+      if (element.status === "Submitted" && element.dty_sub) {
+        liString += `(${element.dty_sub}). `
+      }
+      if (element.title) {
+        liString += `${element.title}. `
```

Comment:
Are you sure there should be a period here? We don't have one after title_secondary.

Reply:
@mf-stevens In the document Nikhil gave us, he explicitly put a period after title and not title_secondary 
https://stevens-ews.atlassian.net/jira/software/projects/EK/issues/EK-125?filter=doneissues&referrer=agility

Reply:
I think that's a mistake in the spec and the "sentence" should start with title and continue on through all the other things until the following period, but for now we'll leave it per the spec.